src/libs/ReportActionsUtils.ts

```diff
@@ -45,6 +57,12 @@ Onyx.connect({
     },
 });
 
+let allPersonalDetails: OnyxEntry<Record<string, PersonalDetails>> = null;
```

Comment:
Please extract to `UserUtils.ts`, and access from `ReportUtils.js` and `ReportActionsUtils.ts` via a utility function like `getPersonalDetails`.